index.html

```diff
@@ -17,7 +17,8 @@ <h1 class="title">INTENTION TIMER</h1>
         <h2 class="subhead">New Activity</h2>
           <section class="category-timer-wrapper">
             <article class="timer-box-wrapper hidden">
-              <h3 class="timer-box-header">Deep Breathing</h3>
+            <!-- <article class="timer-box-wrapper"> -->
+              <h3 class="timer-box-header">Placeholder</h3>
               <div class="countdown-clock">
                 <h1 class="countdown-minutes">00</h1>
```

Comment:
Your site should only have one H1 at a time, so change these additional h1s to spans or p tags

Reply:
We had set each of these elements to h1's in order to get the appropriate styling, but we will revisit this to see if we can find a work-around.